themes/app/src/scss/components/_footer.scss

```diff
@@ -1,26 +1,40 @@
-.color-bar__footer-partners {
-    background-color: $light-sand;
-}
+footer {
+    background-color: $navy-blue;
+    padding: 30px 0;
+
+    p {
```

Comment:
When using BEM we look to avoid targeting `elements` like this, using `classes` is always the advised approach as it gives you greater specificity. The current implementation means that every `paragraph` element you add in the future will now inherit this style, the same with the `img` styles below. So imagine if you added a new `img` element with a completely different format then this would still inherit the generic `img` style under `footer`. The idea is to provide greater specificity so we avoid assumptions that all `paragraph` elements or `img` elements will be the same :)

Reply:
In saying that, for the time being its ok but just for future reference :)

Reply:
Ah, I see. Ok, I'll fix this 😀